src/components/ChatLoader/index.stories.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { ChatLoader } from '.'
+
+export default {
+  title: 'Components/ChatLoader',
+  component: ChatLoader,
+  parameters: {},
+}
+
+const Template = (args) => <ChatLoader {...args} />
+
+export const Default = Template.bind({})
+Default.args = {
+  className: 'typing-loader',
+  height: '8',
+  width: '36',
+  color: 'grey',
+  dotRadius: '4',
+  dotPositions: [
+    { cx: '4', cy: '4' },
```

Comment:
Can we make this as default props so that the we don't have to specify these in our applications?

Reply:
yes, made them as default props.